### PR TITLE
Rename real model test case from VGG-16 to ZFNet

### PR DESCRIPTION
**Pull request by houseroad**
To align with our model zoo.

### CODE REVIEW

File: onnx/backend/test/case/model/__init__.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def collect_testcases():
         ('test_resnet50', 'resnet50'),
         ('test_shufflenet', 'shufflenet'),
         ('test_squeezenet', 'squeezenet'),
-        ('test_vgg16', 'vgg16'),
         ('test_vgg19', 'vgg19'),
+        ('test_zfnet', 'zfnet'),
```
Comment (yinghai) [inline onnx/backend/test/case/model/__init__.py]: what's zfnet? :)

Comment (houseroad) [reply onnx/backend/test/case/model/__init__.py]: Another model for image classification, won ImageNet 2013

Comment (houseroad) [reply onnx/backend/test/case/model/__init__.py]: More details here: https://github.com/onnx/models/tree/master/zfnet

:-)

Comment (jspisak) [reply onnx/backend/test/case/model/__init__.py]: I've never heard of it either. But LGTM.. :)

Comment (houseroad) [reply onnx/backend/test/case/model/__init__.py]: The paper has high citation: 3000+ :-)
Let's read it.

**Pull request merged**